src/docs/product/performance/index.mdx

```diff
@@ -10,23 +10,23 @@ redirect_from:
 description: "Performance Monitoring helps you to see everything from macro-level metrics to micro-level spans, and you'll be able to cross-reference transactions with related issues, and customize queries."
 ---
 
-With performance monitoring, Sentry tracks application performance, measures metrics like throughput and latency, and displays the impact of errors across multiple services. Sentry captures [distributed traces](/product/sentry-basics/tracing/distributed-tracing/) consisting of [transactions and spans](/product/sentry-basics/tracing/distributed-tracing/#traces-transactions-and-spans) to measure individual services and individual operations within those services.
+With <SandboxLink scenario="performance" projectSlug="react">performance monitoring</SandboxLink>, Sentry tracks application performance, measures metrics like throughput and latency, and displays the impact of errors across multiple services. Sentry captures [distributed traces](/product/sentry-basics/tracing/distributed-tracing/) consisting of [transactions and spans](/product/sentry-basics/tracing/distributed-tracing/#traces-transactions-and-spans) to measure individual services and individual operations within those services.
 
 <Note>
 
 Learn more about traces in the [full Tracing documentation](/product/sentry-basics/tracing/).
 
 </Note>
 
-The **Performance** page is the main view in [sentry.io](https://sentry.io) where you can search or browse for transaction data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them.
+The **Performance** page is the main view in [sentry.io](https://sentry.io) where you can search or browse for <SandboxLink scenario="oneTransaction" projectSlug="react">transaction</SandboxLink> data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them.
```

Comment:
I would remove this link here. Generally speaking, if there's a link on a page about a different topic in docs, that link leads to those docs. So in cases where the Sandbox links are embedded in the topic of that page (e.g., in the word "performance" on the page about Performance), I don't think that breaks user expectations. But when it's embedded in the word "transactions" on this page when we have a section of content about transactions in Sentry Basics, it starts to break user expectations.

Reply:
hm... we've gotten a lot of feedback that customers aren't really familiar with transactions. i suggest leaving this here to help with any awareness. and the link is taking them to what a transaction looks like.

Reply:
Can we not link from actual transactions content?